src/apps/frontend/hooks/index.ts

```diff
@@ -0,0 +1,4 @@
+/* eslint-disable import/prefer-default-export */
```

Comment:
Can we either consider updating lint rules or follow the guidelines so we don't have to suppress here and other places?

Reply:
I choose to suppress the warning here because in the future, we might want to export multiple named items from the index.ts file. Currently, we're only exporting a single hook, and TypeScript is reminding us to use the default export. However, if we comply with this warning now, it could lead to unnecessary changes throughout the codebase when we decide to export multiple items later on.

And if we chose to suppress the warning from the TypeScript configuration. This way, we won't be alerted even if we later export just a single item from a file. Which is also not correct.

While generally not a fan of suppressing warnings, in this specific situation, it seems reasonable to do so for the sake of future flexibility and to prevent unnecessary code modifications.

 Interestingly, Baseweb's codebase also includes instances where these TypeScript warnings are suppressed.